templates/lune/data.json

```diff
@@ -0,0 +1,47 @@
+{
+  "lune": {
+    "head-tag": {
+      "extensions": [
+        "amp-sidebar"
+      ],
+      "font-stylesheets": [
+        "https://fonts.googleapis.com/css?family=Lato:400,700,900"
+      ]
+    },
+    "navbar": {
+      "name": "AMP Lune template",
+      "sidebar-id": "header-sidebar",
+      "sidebaronly": true,
+      "href": "lune.amp.html",
+      "logo": {
+         "name": "LUNE",
+         "classes": "h3 xs-hide"
+      },
+      "fixeditems": [
+        {
+          "link": {
+            "url": "mailto:contact@lune.com",
+            "text": "—contact@lune.com"
+          }
+        }
+      ],
+      "nav": [
+        {
+          "link": {"url": "lune.amp.html", "text": "<span>H</span><span>o</span><span>m</span><span>e</span>", "aria-label": "Home"}
+        },
+        {
+          "link": {"url": "product-blanc.amp.html", "text": "<span>B</span><span>l</span><span>a</span><span>n</span><span>c</span>", "aria-label": "Blanc"}
```

Comment:
According to Chrome accessibility tools, the aria-label should override the content, so aria-hidden is not needed on the spans. 

<img width="324" alt="screen shot 2017-07-14 at 12 24 42 pm" src="https://user-images.githubusercontent.com/1872029/28210513-aca22c9e-688f-11e7-811a-39623c9832db.png">

I can confirm by turning on the voice over and navigating through the nav items